src/main/java/com/google/devtools/build/lib/authandtls/GoogleAuthUtils.java

```diff
@@ -53,9 +53,11 @@ public static ManagedChannel newChannel(String target, AuthAndTLSOptions options
     final SslContext sslContext =
         options.tlsEnabled ? createSSlContext(options.tlsCertificate) : null;
 
+    String targetUrl = convertTargetScheme(target);
+
     try {
       NettyChannelBuilder builder =
-          NettyChannelBuilder.forTarget(target)
+          NettyChannelBuilder.forTarget(targetUrl)
               .negotiationType(options.tlsEnabled ? NegotiationType.TLS : NegotiationType.PLAINTEXT)
```

Comment:
implement grpcs in this PR?

Reply:
I'll add it as a next PR, just to keep it separate.

Reply:
it's a one liner

Reply:
not really :)

Reply:
options.tlsEnabled || target.startsWith("grpcs") ? ...